jcl/src/java.base/share/classes/com/ibm/oti/vm/VM.java

```diff
@@ -262,12 +264,27 @@ private VM() {
  * @param loaderType
  *            J9_CLASSLOADER_TYPE_BOOT     - bootstrap classloader
  *            J9_CLASSLOADER_TYPE_PLATFORM - platform classloader
- *            J9_CLASSLOADER_TYPE_OTHERS   - other classloader
+ *            J9_CLASSLOADER_TYPE_APP      - application classloader
+ *            J9_CLASSLOADER_TYPE_OTHERS   - other classloaders
  * @param parallelCapable
  *            true if the loader has registered as parallel capable
  */
 public final static native void initializeClassLoader(ClassLoader loader, int loaderType, boolean parallelCapable);
 
+/*[IF RAM_CLASS_CACHE_SUPPORT] */
+/**
+ * RCP: assign the builtin classloaders(Boot, Platform, Application) in restore run.
```

Comment:
Change this comment to something like:

Link java Classloader to the internal VM classloader. Currently, only Boot, Platform and App classloaders are supported.